lib/jest-helpers.js

```diff
@@ -27,14 +27,29 @@ const nunjucksEnv = nunjucks.configure([configPaths.src, configPaths.components]
  * @returns {String} HTML rendered by the macro
  */
 function renderHtml (componentName, options, callBlock = false) {
+  const macroName = componentNameToMacroName(componentName)
+  const macroPath = `${componentName}/macro.njk`
+
+  return callMacro(macroName, macroPath, [options], callBlock)
+}
+
+/**
+ * Returns the string result from calling a macro
+ *
+ * @param {string} macroName - The name of the macro
+ * @param {string} macroPath - The path to the file containing the macro *from the root of the project*
+ * @param {Array} options - The parameters that will be passed to the macro. They'll be `JSON.stringify`ed and joined with a `,`
```

Comment:
Just a JSDoc suggestion, can we start explaining what it's an array of?

Strings, numbers, objects?

**Slightly better?**

```suggestion
 * @param {object[]} options - The parameters that will be passed to the macro. They'll be `JSON.stringify`ed and joined with a `,`
```

**A lot better?**

```suggestion
 * @param {{ key1: string; key2: number }[]} options - The parameters that will be passed to the macro. They'll be `JSON.stringify`ed and joined with a `,`
```

**Not exactly sure, but flat structure?**
Unless there are some unknown nested objects, can show a flat key/value object too

```suggestion
 * @param {Object<string, string | boolean | number>[]} options - The parameters that will be passed to the macro. They'll be `JSON.stringify`ed and joined with a `,`
```

**Really not sure, but always string keys**
Might have to check this with the TypeScript compiler (might want `unknown` or `any`) but in JSDoc you can say we've always got string keys but any value

```suggestion
 * @param {Object<string, *>[]} options - The parameters that will be passed to the macro. They'll be `JSON.stringify`ed and joined with a `,`
```

Reply:
That one is really an `Array<any>`. Its content will depend on the shape of the macro params that's under test.

```
const macroParams = options.map(param => JSON.stringify(param, null, 2)).join(',')
```

Makes me think the naming is not necessarily ideal, as `options` is usually an object, so I'll swap for `params` which should clarify :)

Reply:
Sorry, there's a `[]` after my `Object<>` examples, "array of objects" shorthand

Yeah it does looks like it's usually (always?) an `object[]` or `undefined`

Reply:
It could be an array of anything, not just object as a macro may accept number, booleans, arrays, objects as any of its parameters.